frameworks/Rust/faf/faf.dockerfile

```diff
@@ -13,4 +13,4 @@ RUN RUSTFLAGS="-Ctarget-cpu=native -Ztune-cpu=native -Zmutable-noalias=yes -Clin
    && strip ./target/x86_64-unknown-linux-gnu/release/faf-ex
 
 EXPOSE 8089
-CMD nice -20 ./target/x86_64-unknown-linux-gnu/release/faf-ex
+CMD nice -19 ./target/x86_64-unknown-linux-gnu/release/faf-ex
```

Comment:
Still not in the rules, but for me this is not fair play.
All the frameworks could do it, but not only 1 fw.
And I'll never use it so in production.

Reply:
It seems a bit off to be commenting on code long unchanged in this pull request